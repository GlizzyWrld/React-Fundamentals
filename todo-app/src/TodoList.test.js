import { render } from '@testing-library/react';
import TodoList from './TodoList';

test('renders TodoList component without errors', () => {
    const { container } = render(<TodoList />);
    expect(container).toMatchSnapshot();
});
