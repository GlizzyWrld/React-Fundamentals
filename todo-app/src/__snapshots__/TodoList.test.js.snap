// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TodoList component without errors 1`] = `
<div>
  <div>
    <form>
      <input
        placeholder="Enter task"
        type="text"
        value=""
      />
      <button
        type="submit"
      >
        Add Todo
      </button>
    </form>
  </div>
</div>
`;
